todoman/ui.py

```diff
@@ -140,12 +143,19 @@ def _save(self, btn=None):
             raise urwid.ExitMainLoop()
 
     def _save_inner(self):
+        self.todo.list = self.current_list
         self.todo.summary = self.summary
         self.todo.description = self.description
         self.todo.location = self.location
+<<<<<<< HEAD
```

Comment:
It appears you tried to resolve the conflict, but you need to remove one of the two "versions" manually (otherwise git could do it automatically). Please remove these added lines.